activesupport/test/cache/stores/mem_cache_store_test.rb

```diff
@@ -268,6 +268,30 @@ def test_can_load_raw_falsey_values_from_dalli_store_with_local_cache
     end
   end
 
+  def test_initial_object_mutation_after_fetch
+    if ActiveSupport::Cache.format_version == 6.1
+      skip "Local cache mutation can't be prevented on legacy MemCacheStore"
```

Comment:
Is this because it returns a raw value instead of an `Entry`? Just wondering why this is the case only for mem cache store.

Reply:
Yeah, it's because in 6.1 mode, the MemCacheStore default coder is basically a noop: https://github.com/rails/rails/blob/acfadacb07f8fbf6900413909519ff920b1ade72/activesupport/lib/active_support/cache/mem_cache_store.rb#L165-L176

It directly passes the entry instance to Dalli, which deals with the marshalling. In theory it's fixable, but it would increase complexity a lot, and that's deprecated code. So I'm not sure it's worth it.